Tables/FE8 Nightmare modules/Class & Character editors/FE8 Class Editor.csv

```diff
@@ -59,7 +59,7 @@ Journeyman (3),0x2d8,0x324,0x38,0x7e,0x2d,0,0x74,0x27,28,8,5,6,5,0,8,6,60,26,29,
 Pupil (3),0x2d9,0x325,0x39,0x7f,0x2e,0,0x7c,0x29,20,5,4,4,5,5,5,6,60,29,28,27,21,26,20,3,55,55,40,35,10,30,40,4,2,0,1,3,3,0x0,0x1,0x0,0x0,0,0,0,0,0,71,1,31,0x88af3d0,0x880ba10,0x880be61,0x880c271,0x880c479,0x880c4ba,0x880c4fb,0x0
 Recruit (3),0x2da,0x332,0x3a,0x37,0x2c,0,0x8a,0x28,23,6,6,11,7,6,6,6,60,23,30,29,22,26,20,3,75,45,40,40,25,35,40,2,2,1,1,2,1,0x40,0x1,0x0,0x0,0,71,0,0,0,0,0,0,0x88af3e4,0x880b849,0x880bc9a,0x880c0aa,0x880c479,0x880c4ba,0x880c4fb,0x0
 Manakete,0x2d7,0x323,0x3b,0x1f,0x2f,0,0xa9,0x1f,20,0,0,2,1,1,25,3,60,20,20,20,20,20,25,3,95,40,30,20,0,10,25,0,0,0,0,0,0,0x0,0x1,0x4,0x0,0,0,0,0,0,0,0,0,0x88af3f0,0x880bb96,0x880bfe7,0x880c3f7,0x880c53c,0x880c57d,0x880c5be,0x0
-Manakete (F),0x2d7,0x323,0x3c,0x20,0x30,0,0x8f,0x1f,12,0,0,2,2,2,5,6,60,20,20,20,20,20,20,2,95,40,30,20,0,10,25,0,0,0,0,0,0,0x0,0x0,0x4,0x0,0,0,0,0,0,0,0,0,0x88af3fc,0x880bb96,0x880bfe7,0x880c3f7,0x880c53c,0x880c57d,0x880c5be,0x0
+Manakete (F),0x2d7,0x323,0x3c,0x20,0x30,0,0x8f,0x1f,20,5,5,3,2,2,10,6,60,20,20,20,20,20,20,2,95,40,30,20,0,10,25,0,0,0,0,0,0,0x0,0x0,0x4,0x0,0,0,0,0,0,0,0,0,0x88af3fc,0x880b90c,0x880bd5d,0x880c16d,0x880c479,0x880c4ba,0x880c4fb,0x0
```

Comment:
What exactly are the changes to the pointers doing?

Reply:
All right. Just if you changed away from Flier, you'd need to change the weapon effectivenesses too (or ask me to do so). If you're reverting it though, no biggie.